src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java

```diff
@@ -136,6 +138,16 @@ public void onGuiChat(ClientChatReceivedEvent e) {
 		String unformatted = Utils.cleanColour(e.message.getUnformattedText());
 		Matcher matcher = SLAYER_XP.matcher(unformatted);
 		if (unformatted.startsWith("You are playing on profile: ")) {
+			if(NotEnoughUpdates.INSTANCE.config.notifications.doBoosterNotif) {
+				try {
+					Field footerField = Minecraft.getMinecraft().ingameGUI.getTabList().getClass().getDeclaredField("field_175255_h");
+					footerField.setAccessible(true);
```

Comment:
Since the field as always the same, you could either use an accessor mixin here, (or in case you don't how to do these, at least save the footerField in a variable instead of recalculating it every time). On the other hand im not sure what the footerField is used for, so maybe it can also just be deleted.

Reply:
I forgot about mixins when I made this, it's using an accessor now